script.skinshortcuts/addon.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.1.0" provider-name="BigNoid, marcelveldt, SiLVO">
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.1.1" provider-name="BigNoid, marcelveldt, SiLVO">
 	<requires>
 		<import addon="xbmc.python" version="3.0.0"/>
 		<import addon="script.module.unidecode" version="0.4.14"/>
+		<import addon="script.module.simpleeval" version="0.0.1"/>
```

Comment:
@enen92 Should I change this to match the version number or it doesn't matter since the one in repo will be higher?

Reply:
yes please use the proper version of the module

Reply:
Ok, Done 👍